phpseclib/Crypt/Base.php

```diff
@@ -2001,11 +2057,48 @@ function _createInlineCryptFunction($cipher_code)
      * for which $mode the lambda function was created.
      *
      * @access private
-     * @return &Array
+     * @return Array &$functions
      */
     function &_getLambdaFunctions()
     {
         static $functions = array();
         return $functions;
     }
+
+    /**
+     * Generates a digest from $bytes
+     *
+     * @see _setupInlineCrypt()
+     * @access private
+     * @param $bytes
+     * @return String
+     */
+    function _trapdoor($bytes)
```

Comment:
generateGUID sounds like taking something from random. This sounds like deterministic digesting/hashing, though.

> It creates loop-unrolled dynamic functions that are specific to the key / block size / number rounds, etc, being used.

_digest_unrolled_function?